# configuration for virtual circle
# circle persist until its life end
float64 radius
float64 x
float64 y
time life